src/EnergyPlus/NodeInputManager.cc

```diff
@@ -109,8 +110,8 @@ namespace NodeInputManager {
 		NumCheckNodes = 0;
 		MaxCheckNodes = 0;
 		NodeVarsSetup = false;
-		NodeWetBulbRepReq.deallocate();
```

Comment:
@rraustad I can put it back.  Or if you do, be sure to pull this branch before working on it.  It's been merged up to current develop and conflicts resolved.  In fact, you should probably review the current diff on github to see if it still looks OK.